File: Python/lt_01_estseq/lt01_estseq_35.py
num1 = int(input("Insira um número Inteiro: "))
num2 = int(input("Insira um número Inteiro: "))
if num2 > num1:
    aux = num1
    num1 = num2
    num2 = aux
SOMA = 0

for i in range(num2 + 1, num1, 1):
    if i % 2 == 1:
        SOMA += i

print("A soma dos Impares entre", num2, "e", num1, "é", SOMA)

File: C/lt_01_estseq/lt01_estseq_35.c
#include <stdio.h>
void mame(int *num1, int *num2) {
  if (*num1 > *num2) {
    int aux = *num1;
    *num1 = *num2;
    *num2 = aux;
  }
}
int soma(int *num1, int *num2) {
  int i, soma = 0;
  for (i = *num1 + 1; i < *num2; i++) {
    if (i % 2 != 0) {
      printf("%d\n", i);
      soma += i;
    }
  }
  return soma;
}
int main() {
  int num1, num2;
  printf("Insira o primeiro numero: ");
  scanf("%d", &num1);

  printf("Insira o segundo numero: ");
  scanf("%d", &num2);

  mame(&num1, &num2);
  printf("A somatoria sera %d.\n", soma(&num1, &num2));
  return 0;
}
